javascript fails syntax checks

Thank you for a very useful plugin.

### What is the current behavior?
yuicompressor syntax-checks the javascript, and finds two issues:
lines 1846 and 1856 use `.class` - "class" is a reserved word.
I changed to use array syntax instead.
````javascript
$header.find('a.' + this.linkInfo[key.replace('Text','')]['class'] + ' span').eq(-1).html(value);
````
Here are the errors:
````
jquery.multiselect.css => jquery.multiselect.min.css
jquery.multiselect.filter.css => jquery.multiselect.filter.min.css
jquery.multiselect.filter.js => jquery.multiselect.filter.min.js
jquery.multiselect.js => jquery.multiselect.min.js
[ERROR] in jquery.multiselect.js
  1846:76:missing name after . operator
[ERROR] in jquery.multiselect.js
  1856:76:missing name after . operator
[ERROR] in jquery.multiselect.js
  1:0:Compilation produced 2 syntax errors.
org.mozilla.javascript.EvaluatorException: Compilation produced 2 syntax errors.
        at com.yahoo.platform.yui.compressor.YUICompressor$1.runtimeError(YUICompressor.java:172)
        at org.mozilla.javascript.Parser.parse(Parser.java:396)
        at org.mozilla.javascript.Parser.parse(Parser.java:340)
        at com.yahoo.platform.yui.compressor.JavaScriptCompressor.parse(JavaScriptCompressor.java:315)
        at com.yahoo.platform.yui.compressor.JavaScriptCompressor.<init>(JavaScriptCompressor.java:536)
        at com.yahoo.platform.yui.compressor.YUICompressor.main(YUICompressor.java:147)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at com.yahoo.platform.yui.compressor.Bootstrap.main(Bootstrap.java:21)
````

### What is the expected behavior?
No errors


### If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo
_Common options for demos are [jsfiddle](https://jsfiddle.net/) and [codepen](https://codepen.io)_

You can get yuicompressor [here](https://github.com/yui/yuicompressor/releases).

To run, just use
````bash
java -jar tools/yuicompressor-2.4.8.jar -o output.min query.multiselect.js
````
Also, please add `*~` to `.gitignore` - these are editor backup files.

### If you are requesting a new or changed feature, please provide a rationale


### Will you submit a corresponding Pull Request to fix the bug or implement the feature?
No.  But it's a trivial work-around.  However, you may want to reconsider using "class" -- 'Class' should work.

Here's the patch (based on HEAD,at commit 6bbf1a6ce2edb6946ca6570f48b11f7e5824458f):
````diff
diff --git a/.gitignore b/.gitignore
index cc8244b..19fe727 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+*~
 docs/public
 docs/_site
 docs/.sass-cache
diff --git a/src/jquery.multiselect.js b/src/jquery.multiselect.js
old mode 100755
new mode 100644
index 257349f..fba982f
--- a/src/jquery.multiselect.js
+++ b/src/jquery.multiselect.js
@@ -1843,7 +1843,7 @@
         case 'expandAllText':
           if (key !== 'checkAllText' || !this.options.maxSelected) {
             // eq(-1) finds the last span
-            $header.find('a.' + this.linkInfo[key.replace('Text','')].class + ' span').eq(-1).html(value);
+            $header.find('a.' + this.linkInfo[key.replace('Text','')]['class'] + ' span').eq(-1).html(value);
           }
           break;
         case 'checkAllIcon':
@@ -1853,7 +1853,7 @@
         case 'expandAllIcon':
           if (key !== 'checkAllIcon' || !this.options.maxSelected) {
             // eq(0) finds the first span
-            $header.find('a.' + this.linkInfo[key.replace('Icon','')].class + ' span').eq(0).replaceWith(value);
+            $header.find('a.' + this.linkInfo[key.replace('Icon','')]['class'] + ' span').eq(0).replaceWith(value);
           }
           break;
         case 'openIcon':
````

### Please tell us about your setup
  
  * Version:
  * Browser:
  * Other frameworks in use:

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+*~
 docs/public
 docs/_site
 docs/.sass-cache

--- a/src/jquery.multiselect.js
+++ b/src/jquery.multiselect.js
@@ -1843,7 +1843,7 @@
         case 'expandAllText':
           if (key !== 'checkAllText' || !this.options.maxSelected) {
             // eq(-1) finds the last span
-            $header.find('a.' + this.linkInfo[key.replace('Text','')].class + ' span').eq(-1).html(value);
+            $header.find('a.' + this.linkInfo[key.replace('Text','')]['class'] + ' span').eq(-1).html(value);
           }
           break;
         case 'checkAllIcon':
@@ -1853,7 +1853,7 @@
         case 'expandAllIcon':
           if (key !== 'checkAllIcon' || !this.options.maxSelected) {
             // eq(0) finds the first span
-            $header.find('a.' + this.linkInfo[key.replace('Icon','')].class + ' span').eq(0).replaceWith(value);
+            $header.find('a.' + this.linkInfo[key.replace('Icon','')]['class'] + ' span').eq(0).replaceWith(value);
           }
           break;
         case 'openIcon':

